diff --git a/lib/index.js b/lib/index.js
index 698efc9357d249cf03adf82a9962499f9455fe66..7f3a8937d781512b3b06f21bbcd91840427f5b71 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -106,1 +106,4 @@ function taskkillTree(pid, force) {
-	], { stdio: "ignore" });
+	], {
+		stdio: "ignore",
+		windowsHide: true
+	});
@@ -746,1 +749,4 @@ function taskkillProcessTree(pid) {
-	], { stdio: "ignore" });
+	], {
+		stdio: "ignore",
+		windowsHide: true
+	});
@@ -805,1 +811,2 @@ function spawnSubprocess(spec, internals = {}) {
-		detached: platform !== "win32"
+		windowsHide: platform === "win32",
+		detached: platform !== "win32"
